src/views/Scan/ViewHandPage.vue

```diff
@@ -133,8 +133,13 @@
             >
               Voeg kaart toe
             </ion-button>
+
             <ion-modal :is-open="isCardModalOpen" trigger="add-card-button">
-              <card-select :current="detections.map(x => x.card)" @card="(card: Card) => { isCardModalOpen = false; addCard(card) }"></card-select>
+              <card-select 
+                :current="detections.map(x => x.card)"
+                @cancel="isModalOpen = false;" 
+                @card="(card: Card) => { isModalOpen = false; addCard(card) }"
+              /></card-select>
```

Comment:
Je merge is niet helemaal goed gegaan. Suggestie zou t moeten fixen
```suggestion
              <card-select 
                :current="detections.map(x => x.card)"
                @cancel="isCardModalOpen = false;" 
                @card="(card: Card) => { isCardModalOpen = false; addCard(card) }"
              />
```